### PR TITLE
Fix ambigious text match in check-docker

### PR DESCRIPTION
**Pull request by stefwalter**
Fix this issue that sometimes happens:

Error: #containers-containers tr:contains("PROBE") is ambigous

### CODE REVIEW

**Comment by dperpeet** [comment]
Looks good! If tests pass, we can merge this.

File: test/check-docker [hunk 1]
```diff
@@ -110,36 +110,37 @@ class TestDocker(MachineCase):
...
         info = json.loads(m.execute('docker inspect PROBE2'))
         self.assertEqual(set(info[0]["HostConfig"]["Links"]),
-                         set(["/PROBE:/PROBE2/alias1", "/PROBE:/PROBE2/alias2"]))
+                         set(["/PROBE1:/PROBE2/alias1", "/PROBE1:/PROBE2/alias2"]))
 
         # delete PROBE2
-        b.wait_in_text('#containers-containers tbody tr:first', "PROBE2")
-        b.wait_not_visible("#containers-containers tbody tr:first button.btn-delete")
+        b.wait_in_text('#containers-containers tbody', "PROBE2")
+        b.wait_not_visible("#containers-containers tbody tr:contains('PROBE2') button.btn-delete")
```
Comment (dperpeet) [inline test/check-docker]: `button.btn-delete` may not be correct

Comment (stefwalter) [inline test/check-docker]: Perhaps, but I don't understand how it could be.

Comment (dperpeet) [inline test/check-docker]: Can't find an error here either, my mistake. I guess we have a race instead. Our preconditions for delete to become available are met before it is disabled again in the ui.

**Comment by dperpeet** [comment]
addressing
```
======================================================================
ERROR: testFrameNavigation (__main__.TestMultiMachine)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "./check-multi-machine", line 65, in setUp
    MachineCase.setUp(self)
  File "/home/hubbot/hubbot/f3cbe0fe9ce25281f956a5bccefa03fff9279831_f22_x86-64/test/testlib.py", line 430, in setUp
    self.machine = self.new_machine()
  File "/home/hubbot/hubbot/f3cbe0fe9ce25281f956a5bccefa03fff9279831_f22_x86-64/test/testlib.py", line 418, in new_machine
    m = self.machine_class(verbose=arg_trace, flavor=flavor, system=system, label="%s-%s" % (program_name, label))
  File "/home/hubbot/hubbot/f3cbe0fe9ce25281f956a5bccefa03fff9279831_f22_x86-64/test/testvm.py", line 626, in __init__
    self.system_connection = libvirt.openReadOnly("qemu:///system")
  File "/usr/lib64/python2.7/site-packages/libvirt.py", line 268, in openReadOnly
    if ret is None:raise libvirtError('virConnectOpenReadOnly() failed')
libvirtError: authentication failed: polkit: Error checking for authorization org.libvirt.unix.monitor: GDBus.Error:org.freedesktop.DBus.Error.NoReply: Message did not receive a reply (timeout by message bus)


----------------------------------------------------------------------
```
in #2908

**Comment by dperpeet** [comment]
docker tests passed, the failures aren't related to the changes in this pull request

**Pull request closed without merging**